js/src/Ice/ConnectionI.js

```diff
@@ -737,7 +732,7 @@ export class ConnectionI {
             this._timer.cancel(this._closeTimeoutId);
         }
 
-        const traceLevels = this._instance.traceLevels();
+        const traceLevels = this._traceLevels;
```

Comment:
We cache _traceLevels and _logger for performance, but the cached fields were not used consistently.